nodewatcher/modules/analysis/rogue_nodes/test_json_files/mainprinter-results.json

```diff
@@ -0,0 +1,673 @@
+[
+    {
+        "name": "04:18:D6:22:58:E4",
+        "probability_being_rogue": 0.5,
+        "ssids": [
+            "CloyneSlow"
+        ]
+    },
+    {
+        "name": "d6b1f709-e6a9-4d5d-a6c9-c796b2de2c1f",
+        "probability_being_rogue": 1,
+        "ssids": [
+            "CloyneSlow",
+            "xfinitywifi",
+            "WHU_STU",
+            "Cloyne"
+        ]
+    },
+    {
+        "name": "04:18:D6:21:58:E4",
+        "probability_being_rogue": 0.0,
+        "ssids": [
+            "Cloyne"
+        ]
+    },
+    {
+        "name": "0C:68:03:E4:4B:23",
+        "probability_being_rogue": 0.0,
+        "ssids": [
+            "eduroam"
+        ]
+    },
+    {
+        "name": "0C:68:03:E4:4B:21",
+        "probability_being_rogue": 0.0,
+        "ssids": [
+            "AirBears2"
+        ]
+    },
+    {
+        "name": "4C:E6:76:AA:2C:E9",
+        "probability_being_rogue": 0.0,
+        "ssids": [
+            "EECS-TEMP-000"
+        ]
+    },
+    {
+        "name": "A0:63:91:D6:83:35",
+        "probability_being_rogue": 0.0,
+        "ssids": [
+            "WhyIsHouseSoTilted?"
+        ]
+    },
+    {
+        "name": "74:D0:2B:67:DA:A6",
+        "probability_being_rogue": 0.0,
+        "ssids": [
+            "domus"
+        ]
+    },
+    {
+        "name": "C0:7C:D1:5D:DB:08",
+        "probability_being_rogue": 0.0,
+        "ssids": [
+            "HOME-707E-2.4"
+        ]
+    },
+    {
+        "name": "04:18:D6:21:58:DF",
+        "probability_being_rogue": 0.0,
+        "ssids": [
+            "Cloyne"
+        ]
+    },
+    {
+        "name": "6C:CA:08:39:18:B0",
+        "probability_being_rogue": 0.0,
+        "ssids": [
+            "f0rest_wOw"
+        ]
+    },
+    {
+        "name": "04:18:D6:21:59:C7",
+        "probability_being_rogue": 0.0,
+        "ssids": [
+            "Cloyne"
+        ]
+    },
+    {
+        "name": "84:61:A0:E8:F0:50",
+        "probability_being_rogue": 0.0,
+        "ssids": [
+            "ATT576"
+        ]
+    },
+    {
+        "name": "24:A4:3C:2A:28:60",
+        "probability_being_rogue": 0.0,
+        "ssids": [
+            "JSTJEsuits"
+        ]
+    },
+    {
+        "name": "04:18:D6:21:59:C8",
+        "probability_being_rogue": 0.0,
+        "ssids": [
+            "Cloyne"
+        ]
+    },
+    {
+        "name": "5E:8F:E0:AA:C7:A0",
+        "probability_being_rogue": 0.0,
+        "ssids": [
+            null
+        ]
+    },
+    {
+        "name": "38:EA:A7:39:09:93",
+        "probability_being_rogue": 0.5,
+        "ssids": [
+            "HP-Print-93-Photosmart 6520"
+        ]
+    },
+    {
+        "name": "62e3b05f-e0b9-4c4d-9961-ceba4a3df787",
+        "probability_being_rogue": 1,
+        "ssids": [
+            "CloyneSlow",
+            "eduroam",
+            "JSTJEsuits",
+            null,
+            "Allenoke-2.4",
+            "TP-LINK_C57684"
```

Comment:
Is this another node? But why it looks registered/known? But probability to be rogue is 1? How can a known node (with UUID) have 1 as probability?